openapi/definitions.yaml

```diff
@@ -1065,4 +1084,98 @@ MessageCategoryPayment:
       type: string
   required:
     - tag
-    - rptId
\ No newline at end of file
+    - rptId
+LegalMessage:
+  type: object
+  properties:
+    eml: 
+      $ref: "#/LegalMessageEml"
+    cert_data:
+      $ref: "#/LegalMessageCertData"
+  required: 
+    - eml
+    - cert_data
+LegalMessageEml:
+  type: object
+  properties:
+    subject:
+      type: string
+    plain_text_content:
+      type: string
+    html_content:
+      type: string
+    attachments:
+      type: array
+      items:
+        $ref: '#/Attachment'
+  required:
+    - subject
+    - plain_text_content
+    - html_content
+    - attachments
+LegalMessageCertData:
+  type: object
+  properties:
+    header:
+      $ref: "#/CertDataHeader"
+    data:
+      $ref: "#/CertData"
+  required:
+    - header
+    - data
+CertData:
+  type: object
+  properties:
+    sender_provider:
+      type: string
+      minLength: 1
+    timestamp:
+      type: string
+      format: date-time
+    envelope_id:
+      type: string
+      minLength: 1
+    msg_id:
+      type: string
+      minLength: 1
+    receipt_type:
+      type: string
+  required:
+    - sender_provider
+    - timestamp
+    - envelope_id
+    - msg_id
+    - receipt_type
+CertDataHeader:
+  type: object
+  properties:
+    sender:
+      type: string
+      minLength: 1
+    recipients:
+      type: string
+      minLength: 1
+    replies:
+      type: string
+      minLength: 1
```

Comment:
maybe these properties should be more structured as array of recipient and array of reply